### ISSUE TITLE
Have a roadmap to re-review existing affiliated packages

### ISSUE THREAD

**Issue by pllim** [astropy/astropy-project#139]
We have collected a number of [affiliated packages](https://www.astropy.org/affiliated/#affiliated-package-list) over the years (42 at the time of writing this). To ensure that the quality of being an affiliated package is maintained, there is a need to go back and re-review them.

Possible outcome for an existing affiliated package:

* Stay affiliated: If badges remain the same or have improved overall.
* Get unaffiliated (after a grace period for the package to try to fix problems): If the package is outright abandoned, or badges have worsened overall, or any of the badges turns red.

cc @hamogu and @astropy/coordinators

**Comment by astrofrog** [comment]
I agree with this!

**Comment by pllim** [comment]
Sorted the existing affiliated packaged (not coordinated) by last updated. Is this a good place to start, by going down this list?

  <table>
   <thead>
    <tr>
     <th>name</th>
     <th>repo_url</th>
     <th>last_updated</th>
    </tr>
   </thead>
   <tr>
    <td>Saba</td>
    <td>https://github.com/astropy/saba</td>
    <td>2017-09-26</td>
   </tr>
   <tr>
    <td>galpy</td>
    <td>https://github.com/jobovy/galpy</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>synphot</td>
    <td>https://github.com/spacetelescope/synphot_refactor</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>hips</td>
    <td>https://github.com/hipspy/hips</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>poliastro</td>
    <td>https://github.com/poliastro/poliastro</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>linetools</td>
    <td>https://github.com/linetools/linetools</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>pyspeckit</td>
    <td>https://github.com/pyspeckit/pyspeckit</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>marxs</td>
    <td>https://github.com/Chandra-MARX/marxs</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>cluster-lensing</td>
    <td>https://github.com/jesford/cluster-lensing</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>Halotools</td>
    <td>https://github.com/astropy/halotools</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>stingray</td>
    <td>https://github.com/StingraySoftware/stingray</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>HENDRICS</td>
    <td>https://github.com/StingraySoftware/HENDRICS</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>gala</td>
    <td>https://github.com/adrn/gala</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>naima</td>
    <td>https://github.com/zblz/naima</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>Astro-SCRAPPY</td>
    <td>https://github.com/astropy/astroscrappy</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>omnifit</td>
    <td>https://github.com/RiceMunk/omnifit</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>spectral-cube</td>
    <td>https://github.com/radio-astro-tools/spectral-cube</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>ginga</td>
    <td>https://github.com/ejeschke/ginga</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>PyDL</td>
    <td>https://github.com/weaverba137/pydl</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>APLpy</td>
    <td>https://github.com/aplpy/aplpy</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>astroplan</td>
    <td>https://github.com/astropy/astroplan</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>gammapy</td>
    <td>https://github.com/gammapy/gammapy</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>Glue</td>
    <td>https://github.com/glue-viz/glue</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>sncosmo</td>
    <td>https://github.com/sncosmo/sncosmo</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>imexam</td>
    <td>https://github.com/spacetelescope/imexam</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>spherical_geometry</td>
    <td>https://github.com/spacetelescope/spherical_geometry</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>gwcs</td>
    <td>https://github.com/spacetelescope/gwcs</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>pyregion</td>
    <td>https://github.com/astropy/pyregion</td>
    <td>2017-11-08</td>
   </tr>
   <tr>
    <td>dust_extinction</td>
    <td>https://github.com/karllark/dust_extinction</td>
    <td>2018-05-25</td>
   </tr>
   <tr>
    <td>corral</td>
    <td>https://github.com/toros-astro/corral</td>
    <td>2018-08-21</td>
   </tr>
   <tr>
    <td>feets</td>
    <td>https://github.com/carpyncho/feets</td>
    <td>2018-11-27</td>
   </tr>
   <tr>
    <td>baseband</td>
    <td>https://github.com/mhvk/baseband</td>
    <td>2018-11-27</td>
   </tr>
   <tr>
    <td>sbpy</td>
    <td>https://github.com/NASA-Planetary-Science/sbpy</td>
    <td>2019-06-18</td>
   </tr>
   <tr>
    <td>EinsteinPy</td>
    <td>https://github.com/einsteinpy/einsteinpy</td>
    <td>2019-07-23</td>
   </tr>
   <tr>
    <td>astroalign</td>
    <td>https://github.com/toros-astro/astroalign</td>
    <td>2019-08-15</td>
   </tr>
   <tr>
    <td>astroML</td>
    <td>https://github.com/astroML/astroML</td>
    <td>2019-10-06</td>
   </tr>
   <tr>
    <td>PyVO</td>
    <td>https://github.com/astropy/pyvo</td>
    <td>2019-10-06</td>
   </tr>
   <tr>
    <td>mocpy</td>
    <td>https://github.com/cds-astro/mocpy</td>
    <td>2019-11-04</td>
   </tr>
   <tr>
    <td>statmorph</td>
    <td>https://github.com/vrodgom/statmorph</td>
    <td>2020-01-21</td>
   </tr>
   <tr>
    <td>ligo.skymap</td>
    <td>https://git.ligo.org/lscsoft/ligo.skymap</td>
    <td>2020-07-01</td>
   </tr>
   <tr>
    <td>agnpy</td>
    <td>https://github.com/cosimoNigro/agnpy</td>
    <td>2020-07-13</td>
   </tr>
   <tr>
    <td>lenstronomy</td>
    <td>https://github.com/sibirrer/lenstronomy</td>
    <td>2020-07-24</td>
   </tr>
  </table>

p.s. Found a bug! astropy/astropy#10823

**Comment by adrn** [comment]
What does "last updated" mean"

**Comment by pllim** [comment]
Not sure... Maybe CoCo knows?

**Comment by adrn** [comment]
I only ask because it's definitely not the date that each package was last updated (gala has a release from July 2020 and commits from days ago), so it must be a log of when the affiliated package record was last updated..?

**Comment by adrn** [comment]
Oh yea, they seem to be static values from the [affiliated package JSON file](https://www.astropy.org/affiliated/registry.json) - sorry for the noise!

**Comment by pllim** [comment]
Looks like @bsipocz explained it at https://github.com/astropy/astropy.github.com/pull/382/files#r448593260

**Comment by astrofrog** [comment]
Yes just confirming it is when the review results were last updated. We could rename the key to include review perhaps?

**Comment by eteq** [comment]
I also agree with doing this and confirm that that date was intended for exactly the purpose here (to know when to re-review).

That said... I feel like there's an argument for trying to hold the number at 42... :wink:

**Comment by pllim** [comment]
@hamogu , how about we plan to start the cycle, say Jan 2021, and strive to re-review one package every 2 months?

**Comment by astrofrog** [comment]
7 years assuming no new packages!

**Comment by pllim** [comment]
We could try parallel processing but will need more cores. 🤓

**Comment by astrofrog** [comment]
Hopefully would work better than the io.ascii parallel reader?

**Comment by pllim** [comment]
Also see #334 . I think we're leaning towards automation and not a full re-review.
